CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ All notable changes to this project will be documented in this file.
     endpoint if it was not provided, falling back to `us-east-2`.
   - The default is now `us-east-1`. Please set the region explicitly if you are using a different
     one.
+- Bump `stackable-versioned` to 0.6.0 ([#722]).
```

Comment:
nit: According to architecture decision line 334 we don't track those

Reply:
My opinion on this is still the same. I added some more reasoning over here: https://github.com/stackabletech/airflow-operator/pull/591#discussion_r1995260955